tests/configs/default.js

```diff
@@ -1,8 +1,3 @@
-/* Magic Mirror Test config sample environment
- *
- * By Rodrigo Ramírez Norambuena https://rodrigoramirez.com
- * MIT Licensed.
- */
```

Comment:
I'd keep some kind of comment header so that people know what this default js does

Reply:
The comments have been lost, will paste them back.